shared/actions/platform-specific/index.native.tsx

```diff
@@ -507,14 +507,17 @@ async function manageContactsCache(
 
   // feature enabled and permission granted
   const contacts = await Contacts.getContactsAsync()
+  let defaultCountryCode: string
+  try {
```

Comment:
cc @jakob223 @jzila I did some shuffling here to handle the potential `reject` in the android module